### PR TITLE
corrected small bugs in template creation

### PR DESCRIPTION
**Pull request by ggoffy**

### CODE REVIEW

**Comment by txmodxoops** [comment]
@ggoffy 

Can you help me write all the html code or heredoc that still exists in some file and turn it into php with the functions that I have created myself?

Thank you!

**Comment by ggoffy** [comment]
hi timgno

can you explain, what you mean?

**Comment by ggoffy** [comment]
do you have my email?

**Comment by txmodxoops** [comment]
Hi goffy

I don't have your email!

What I wanted to tell you, it is that you have edited the template files and you've noticed that I have re-written the code yet.

All that code is rewritten with php functions.

If you look inside the file TemplatesUserIndex.php, it has already rewritten on getTemplatesUserIndexTableTbody function

That's how it should be written the code.

If you have new ideas you can use them as you wish

Thanks for your time

File: class/files/user/UserPages.php [hunk 1]
```diff
@@ -171,7 +171,7 @@ private function getUserPagesFooter($moduleDirname, $tableName, $tableSoleName,
         $ret .= $uc->getUserMetaKeywords($moduleDirname);
         $ret .= $pc->getPhpCodeUnset('keywords');
         $ret .= $pc->getPhpCodeCommentLine('Description');
-        $ret .= $uc->getUserMetaDesc($moduleDirname, $language, $stuTableSoleName);
```
Comment (txmodxoops) [inline class/files/user/UserPages.php:174]: This was right, why did you change it?

Comment (ggoffy) [inline class/files/user/UserPages.php:174]: e.g. table name is 'clients' , table sole name is 'client'
with `$ret .= $uc->getUserMetaDesc($moduleDirname, $language, $stuTableSoleName);` you get `modulenameMetaDescription(_MA_MODULENAME_CLIENT_DESC);`
but in the language file you have created _MA_MODULENAME_CLIENTS_DESC

so, one of them must be changed

**Comment by ggoffy** [comment]
now you have it ;-)

Am 21.06.2016 um 11:19 schrieb Txmod Xoops:

> Hi goffy
> 
> I don't have your email!
